src/main/java/com/tterrag/blur/mixin/MixinScreen.java

```diff
@@ -36,14 +36,14 @@ public abstract class MixinScreen {
     }
 
     @ModifyConstant(
-            method = "renderBackground(Lnet/minecraft/client/util/math/MatrixStack;I)V",
+            method = "Lnet/minecraft/class_437;method_25420(Lnet/minecraft/class_4587;)V",
```

Comment:
Is there a reason to use the Intermediary Mappings?
It is present in yarn:
https://linkie.shedaniel.me/mappings?namespace=yarn&version=1.19.4&search=method_25420

Reply:
While I testing, the MInecraft say that part has error while on lunch.
So, I trace the logs, that part is IDEA can't find method. So I changed that part intermediary mapping, the problem was solved.